app/src/main/java/org/simple/clinic/util/DistinctValueCallback.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.util
```

Comment:
This class is being used in the previous commit. Can you either:
- Reorder it to be before the previous commit?
- Squash it with the previous commit?